app/components/main-content/PrelaunchSignUpForm.tsx

```diff
@@ -61,87 +61,105 @@ const PrelaunchSignUpForm: React.FC<PrelaunchSignUpFormProps> = ({
+        >
+          {textInputConfig.map((input, index) =>
+            input.tooltipLabel ? (
+              <Tooltip
+                multiline
+                key={`${input.field}-${index}`}
+                label={input.tooltipLabel}
+                position="bottom-start"
+                styles={{
+                  tooltip: {
+                    backgroundColor: '#3D5B50',
+                    color: 'white',
+                    padding: '8px 12px',
+                    borderRadius: '4px',
+                    position: 'absolute',
+                    width: '300px',
+                    fontSize: '11px',
+                  },
+                }}
+              >
                 <TextInput
-                  key={`${input.field}-${index}`}
-                  id={`${input.field}-input`}
                   classNames={{
                     root: classes.root,
-                    input: form.errors[input.field] ? `${classes.input} ${classes.inputError}` : classes.input,
+                    input: classes.input,
                     label: classes.label,
-                    error: classes.error
+                    error: classes.error,
                   }}
-                  error={form.errors[input.field]}
                   label={input.label}
                   required={input.required}
                   aria-required={input.required ? 'true' : 'false'}
                   {...form.getInputProps(input.field)}
                 />
-              )
-            )}
-            <section className='text-center mt-10'>
-              <UniversalButton
-                id="join-waiting-list-button"
-                type="submit"
-                label={isLoading ? "Adding..." : "Join Waiting List"}
-                aria-label={isLoading ? "Adding you to our waiting list" : "Join Waiting List"}
+              </Tooltip>
+            ) : (
+              <TextInput
+                key={`${input.field}-${index}`}
+                id={`${input.field}-input`}
                 classNames={{
-                  root: 'button-primary button-accent',
+                  root: classes.root,
+                  input: form.errors[input.field]
+                    ? `${classes.input} ${classes.inputError}`
+                    : classes.input,
+                  label: classes.label,
+                  error: classes.error,
                 }}
-                disabled={isLoading}
+                error={form.errors[input.field]}
+                label={input.label}
+                required={input.required}
+                aria-required={input.required ? 'true' : 'false'}
+                {...form.getInputProps(input.field)}
               />
-
-              <footer className="pt-2 mb-2 text-light-text">
-                <PrivacyAgreement />
-              </footer>
-            </section>
-          </section>
-        </form>
-      </Box>
-    </>
+            )
+          )}
+          <div className="mt-10 text-center">
```

Comment:
Lines 126-146 contain the actual changes. It may seem like there are more changes than there actually are because I removed the redundant fragment outside of the box component. I also removed the footer component and replaced it with a simple Linke component inside the checkbox. 

I initially used the Checkbox component from Mantine, but it conflicted with our Tailwind configuration and generated a bug (a huge tick icon appeared and I needed to hide it with styling). Also, I was unable to align the checkbox and label horizontally within a reasonable timeframe, which is why I switched back to the basic checkbox input to save time.